spec/tasks/broadcasts_spec.rb

```diff
@@ -0,0 +1,40 @@
+require "rails_helper"
+
+RSpec.describe "Broadcasts tasks", type: :task do
+  let(:service)         { Broadcasts::WelcomeNotification::Generator }
+  let(:next_week_today) { 1.week.since }
+
+  let_it_be_readonly(:default_config_date) { SiteConfig.welcome_notifications_live_at }
```

Comment:
```suggestion
  let_it_be_readonly(:default_config_date) { SiteConfig.welcome_notifications_live_at } 
```
If the date defaults to empty/`nil`, should we note that here?

Reply:
I think it's fine right now. The spec assumes that that variable is set and has dedicated a spec for the scenario where that variable is nil.